src/java.cpp

```diff
@@ -14,7 +14,8 @@
 
 #define DYNAMIC_PROXY_JS_ERROR -4
 
-long v8ThreadId;
+long v8ThreadIdWIN32;
```

Comment:
Could this be replaced with

```
#ifdef WIN32
  typedef long v8ThreadId;
#else
  typedef pthread_t v8ThreadId;
#endif
```
and change the usages to use the new type `v8ThreadId`?